### ISSUE TITLE
Describir técnica estadística para la frecuencia de variables.

### ISSUE THREAD

**Issue by unintendedbear** [geneura-papers/2014_CS_Journal_URLS#11]
Supongo que ya os habrá salido, pero que sepáis que en https://docs.google.com/spreadsheets/d/1wzcXjYKUAXcQlDcOwNwtavJ-ayByOcRSm5jPbyxmLE0/edit?usp=sharing estoy poniendo los resultados de parsear las reglas que salen para cada distribución 80-20 (en total 10, llevo 3, es muy tedioso y PART tarda siglo y medio en ejecutarse).
En Dropbox además están todos los logs que voy sacando y todas las distribuciones aleatorias (JJ, creo que tienes permisos para entrar en esa carpeta, pero bueno como es Dropbox y son muchos CSV y ARFF, tampoco te he insistido con ello).

Por fa, si podéis, pasadme una descripción de la técnica estadística que vayamos a usar, o ponedla en el paper, como queráis. O bien decidme cuál y yo busco información y lo escribo.

Muchas gracias.

**Comment by amorag** [comment]
Ok, gracias.

Yo me ocupo de la estadística.

Un saludo.

De: Paloma de las Cuevas Delgado [mailto:notifications@github.com] 
Enviado el: jueves, 15 de enero de 2015 11:01
Para: geneura-papers/2014_CS_Journal_URLS
CC: Antonio Mora
Asunto: [2014_CS_Journal_URLS] Describir técnica estadística para la frecuencia de variables. (#11)

Supongo que ya os habrá salido, pero que sepáis que en https://docs.google.com/spreadsheets/d/1wzcXjYKUAXcQlDcOwNwtavJ-ayByOcRSm5jPbyxmLE0/edit?usp=sharing estoy poniendo los resultados de parsear las reglas que salen para cada distribución 80-20 (en total 10, llevo 3, es muy tedioso y PART tarda siglo y medio en ejecutarse).
En Dropbox además están todos los logs que voy sacando y todas las distribuciones aleatorias (JJ, creo que tienes permisos para entrar en esa carpeta, pero bueno como es Dropbox y son muchos CSV y ARFF, tampoco te he insistido con ello).

Por fa, si podéis, pasadme una descripción de la técnica estadística que vayamos a usar, o ponedla en el paper, como queráis. O bien decidme cuál y yo busco información y lo escribo.

Muchas gracias.

—
Reply to this email directly or view it on GitHub https://github.com/geneura-papers/2014_CS_Journal_URLS/issues/11 .  https://github.com/notifications/beacon/AC6-WnCERIZg1aHCCmsF6_oVPhPcBaq2ks5nh4dzgaJpZM4DSu7-.gif 

No se encontraron virus en este mensaje.
Comprobado por AVG - www.avg.com
Versión: 2013.0.3495 / Base de datos de virus: 4257/8932 - Fecha de publicación: 01/14/15

**Comment by JJ** [comment]
Sí, me va apareciendo por Dropbox. A ver si tengo tiempo más adelante,
ahora tengo tres o cuatro cosas (hasta viernes por la tarde)

El 15 de enero de 2015, 11:40, Antonio Mora notifications@github.com
escribió:

> Ok, gracias.
> 
> Yo me ocupo de la estadística.
> 
> Un saludo.
> 
> De: Paloma de las Cuevas Delgado [mailto:notifications@github.com]
> Enviado el: jueves, 15 de enero de 2015 11:01
> Para: geneura-papers/2014_CS_Journal_URLS
> CC: Antonio Mora
> Asunto: [2014_CS_Journal_URLS] Describir técnica estadística para la
> frecuencia de variables. (#11)
> 
> Supongo que ya os habrá salido, pero que sepáis que en
> https://docs.google.com/spreadsheets/d/1wzcXjYKUAXcQlDcOwNwtavJ-ayByOcRSm5jPbyxmLE0/edit?usp=sharing
> estoy poniendo los resultados de parsear las reglas que salen para cada
> distribución 80-20 (en total 10, llevo 3, es muy tedioso y PART tarda siglo
> y medio en ejecutarse).
> En Dropbox además están todos los logs que voy sacando y todas las
> distribuciones aleatorias (JJ, creo que tienes permisos para entrar en esa
> carpeta, pero bueno como es Dropbox y son muchos CSV y ARFF, tampoco te he
> insistido con ello).
> 
> Por fa, si podéis, pasadme una descripción de la técnica estadística que
> vayamos a usar, o ponedla en el paper, como queráis. O bien decidme cuál y
> yo busco información y lo escribo.
> 
> Muchas gracias.
> 
> —
> Reply to this email directly or view it on GitHub <
> https://github.com/geneura-papers/2014_CS_Journal_URLS/issues/11> . <
> https://github.com/notifications/beacon/AC6-WnCERIZg1aHCCmsF6_oVPhPcBaq2ks5nh4dzgaJpZM4DSu7-.gif>
> 
> No se encontraron virus en este mensaje.
> Comprobado por AVG - www.avg.com
> Versión: 2013.0.3495 / Base de datos de virus: 4257/8932 - Fecha de
> publicación: 01/14/15
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/geneura-papers/2014_CS_Journal_URLS/issues/11#issuecomment-70067904
> .

## 

JJ
